src/libs/actions/OnyxUpdates.ts

```diff
@@ -132,29 +132,23 @@ function saveUpdateInformation(updateParams: OnyxUpdatesFromServer) {
  * This function will receive the previousUpdateID from any request/pusher update that has it, compare to our current app state
  * and return if an update is needed
  * @param previousUpdateID The previousUpdateID contained in the response object
+ * @param clientLastUpdateID an optional override for the lastUpdateIDAppliedToClient
```

Comment:
What's the purpose of this change?

Reply:
I explained in the description above. In the background, `Onyx.connect` in this module doesn't run so we need to pass the value for `lastUpdateIDAppliedToClient` as a param.